client/src/components/RoundAddDialog.vue

```diff
@@ -22,7 +22,7 @@
             />
           </v-radio-group>
         </v-col>
-        <v-col cols="6">
+        <v-col cols="6" v-if="!game.coop">
```

Comment:
It should be !team.coop
The logic of coop is: The whole game can be cooperative e.g. Arkham Horror
The team in the game can be cooperative e.g. Star Realms have cooperative mode. 
So team is more specific

Reply:
I've created a discord channel. we can discuss app logic there as the future view of the app. I've send you an invitation